DataRepo/templates/DataRepo/compound_list.html

```diff
@@ -1,11 +1,16 @@
 {% extends "base.html" %}
+{% load static %}
 {% load customtags %}
 
 {% block title %}Compounds{% endblock %}
 
+{% block head_extras %}
+    <link href="{% static 'css/bootstrap_table_cus1.css' %}" rel="stylesheet">
+{% endblock %}
+
 {% block content %}
-<div>
-    <h4>List of Compounds</h4>
+<h4>List of Compounds</h4>
```

Comment:
nb. To be more succinct...
```suggestion
<h4>Compounds List</h4>
```